tests/e2e/default/targets/target-aggregates.wdio-spec.js

```diff
@@ -109,317 +49,309 @@ describe('Target aggregates', () => {
+    
+    const userWithManyPlaces = userSettingsFactory.build({
       _id: 'org.couchdb.user:offline_many_facilities',
-      language: 'en',
-      known: true,
-      type: 'user-settings',
+      name: 'offline_many_facilities',
       roles: [ 'chw' ],
-      facility_id: [ parentPlace._id, otherParentPlace._id ],
-      contact_id: contactWithManyPlaces._id,
-      name: 'offline_many_facilities'
-    };
+      facility_id: [ districtHospital1._id, districtHospital2._id ],
+      contact_id: contactWithManyPlaces._id
+    });
     const userWithManyPlacesPass = uuid();
 
-    before(async () => {
-      const allDocs = [ ...docs, parentPlace, otherParentPlace, contactWithManyPlaces, userWithManyPlaces ];
-      await utils.saveDocs(allDocs);
-      await utils.createUsers([user]);
-      await utils.request({
-        path: `/_users/${userWithManyPlaces._id}`,
-        method: 'PUT',
-        body: { ...userWithManyPlaces, password: userWithManyPlacesPass, type: 'user' },
+    const contactDocs = [];
+    const targetDocs = [];
+    Array
+      .from({ length: 5 })
+      .forEach((e, i) => {
+        const district1Data = helperFunctions.generateContactsAndTargets(
+          districtHospital1, NAMES_DH1[i], TARGET_VALUES_BY_CONTACT,
+        );
+        const district2Data = helperFunctions.generateContactsAndTargets(
+          districtHospital2, NAMES_DH2[i], TARGET_VALUES_BY_CONTACT,
+        );
+
+        contactDocs.push(...district1Data.contacts, ...district2Data.contacts);
+        targetDocs.push(...district1Data.targets, ...district2Data.targets);
       });
-      await browser.url('/medic/login');
-    });
 
     const DOCS_TO_KEEP = [
-      parentPlace._id,
-      otherParentPlace._id,
-      ...docs.map(doc => doc._id),
+      districtHospital1._id,
+      districtHospital2._id,
+      ...contactDocs.map(doc => doc._id),
       'fixture:user:supervisor',
       'org.couchdb.user:supervisor',
       '^target~',
       [/^form:/],
     ];
 
+    before(async () => {
+      await utils.saveDocs([
+        ...contactDocs, districtHospital1, districtHospital2, contactWithManyPlaces, userWithManyPlaces
+      ]);
+      await utils.createUsers([onlineUser]);
+      await utils.request({
+        path: `/_users/${userWithManyPlaces._id}`,
+        method: 'PUT',
+        body: { ...userWithManyPlaces, password: userWithManyPlacesPass, type: 'user' },
+      });
+      await browser.url('/medic/login');
+    });
+
     afterEach(async () => {
       await commonPage.logout();
       await utils.revertDb(DOCS_TO_KEEP, true);
     });
 
-    it('should disable content when user has many facilities associated', async () => {
-      await loginPage.login({ password: userWithManyPlacesPass, username: userWithManyPlaces.name });
-      await commonPage.waitForPageLoaded();
-      await targetAggregatesPage.checkContentDisabled();
-    });
+    describe('Online user with one place associated', () => {
```

Comment:
Is it the same behavior for an `offline` user with only one place associated?
If it is not, we should add the scenario.

Reply:
Yes it is. The different behavior depends on whether the user has one or more than one place associated.

Reply:
Correct. It's based on associated places to a user

Reply:
Yes Tatiana you are correct. The difference in behavior is whether user has one place or more than one place.